webpack.config.js

```diff
@@ -26,6 +26,23 @@ const SENTRY_WEBPACK_PROXY_PORT = env.SENTRY_WEBPACK_PROXY_PORT;
 const USE_HOT_MODULE_RELOAD =
   !IS_PRODUCTION && SENTRY_BACKEND_PORT && SENTRY_WEBPACK_PROXY_PORT;
 
+// Deploy previews are built using netlify. We can check if we're in netlifys
+// build process by checking the existance of the PULL_REQUEST env var.
+//
+// See: https://www.netlify.com/docs/continuous-deployment/#environment-variables
+const DEPLOY_PREVIEW_CONFIG = env.PULL_REQUEST && {
```

Comment:
Will `env.PULL_REQUEST` conflict with travis env vars?

Reply:
luckily unlike netlify, travis prefixes their env vars. So on travis it's `TRAVIS_PULL_REQUEST`.